### DISCUSSION TITLE
Deno-Deploy app page don't load!

### DISCUSSION THREAD

**Discussion by andres-rod-g** [community/community#34880] [Other Feature Feedback, Questions, & Ideas]
Hey! Deno-deploy GitHub app is not loading! 

Link: 
https://github.com/apps/deno-deploy/installations/new?state=link

![image](https://user-images.githubusercontent.com/79536152/193467512-c7f6f4a6-e6f2-4ded-b855-0e24a9015f27.png)

**Comment by queenofcorgis** [comment]
Have you tried clearing your cookies/cache or using a different browser/device?

**Reply by andres-rod-g** [reply]
Hey!

Yeas, I tried, and nothing of that worked. The only thing that worked was removing alm GitHub apps from my account, quit from all the organizations I was in. I think my account was kind of glitched 😅
